test/net/DialogTypeLocators/NamingConventionDialogTypeLocatorTest.cs

```diff
@@ -32,9 +31,6 @@ public NamingConventionDialogTypeLocatorTest()
             assemblyBuilder.CreateType("TestAssembly.ModuleWithoutViewNamespace.ViewModel.DialogViewModel", typeof(ViewModelBase));
 
             assemblyBuilder.CreateType("TestAssembly.UnconventionalNamespace.DialogViewModel", typeof(ViewModelBase));
-
-            // Create assembly
-            testAssembly = assemblyBuilder.Build();
```

Comment:
This is not available for .Net Core and above. Aimed for .Net 7, but the tests don't even require an Assembly object. AssemblyBuilder is probably just as good.

Reply:
You are correct, we can remove this code